test/integration/app/prompted_nobuild.js

```diff
@@ -326,18 +326,19 @@ describe('Prompt and no build integration tests for app generator', function ()
                             name: 'notes',
                             dir: 'notes',
                             appPattern: 'Basic',
-                            services: ['Cloudant', 'Redis', 'Object Storage', 'AppID', 'Watson Conversation'],
-                            configure: ['Cloudant / CouchDB', 'Redis', 'Object Storage', 'AppID', 'Watson Conversation'],
-                            cloudantName: 'testCloudant',
+                            services: ['Cloudant', 'Redis', 'Object Storage', 'AppID', 'Watson Conversation', 'Alert Notification'],
+                            configure: ['Cloudant / CouchDB', 'Redis', 'Object Storage', 'AppID', 'Watson Conversation', 'Alert Notification'],
+                           cloudantName: 'testCloudant',
```

Comment:
There's a minor whitespace change here that looks unintended. Would be nice to clean that up, but shouldn't block merging this.